plugins/modules/swim_workflow_manager.py

```diff
@@ -34,6 +34,16 @@
     description: Set to True to verify the Cisco Catalyst Center config after applying the playbook config.
     type: bool
     default: False
+  max_timeout:
+    description: Sets the maximum duration in seconds for attempting to retrieve task details from the API. If the
+        task details are not obtained within this timeframe, the loop will be terminated, and a log message will be
+        generated indicating the timeout.
```

Comment:
Can we say like the process will be terminated instead ofsaying  "loop will be terminated"? Because it is more code oriented.. For example, if we rename to "dnac_api_task_timeout" then:

`description: Defines the timeout in seconds for API calls to retrieve task details. If the task details are not received within this period, the process will end, and a timeout notification will be logged. `

Reply:
added.